src/contract-state.ts

```diff
@@ -69,9 +69,8 @@ export type GatewayStakingSettings = {
 
 export type GatewayMetadata = {
   label: string;
-  note?: string;
-  properties?: string;
-  observerWallet?: WalletAddress;
+  note: string;
+  properties: string;
```

Comment:
These properties are required to join the network and are non nullable in gateway settings updates